.travis.yml

```diff
@@ -2,14 +2,17 @@ language: node_js
 node_js:
   - "stable"
 sudo: false
-env: REACT_VERSION=15.3.x
+env: REACT_VERSION=15.5.x
```

Comment:
Ah, yea I had seen that prior when you linked. Am new(ish) still to travis so not sure on best practices. I'll take a look and give this a shot in the future.